everestrs/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+include(cxxrs)
+
+find_program(CXXBRIDGE cxxbridge PATHS "$ENV{HOME}/.cargo/bin/")
+if (CXXBRIDGE STREQUAL "CXXBRIDGE-NOTFOUND")
+  message("Could not find cxxbridge, trying to install with `cargo install cxxbridge-cmd'")
+  find_program(CARGO cargo PATHS "$ENV{HOME}/.cargo/bin/")
+  if (CARGO STREQUAL "CARGO-NOTFOUND")
+    message(FATAL_ERROR "Requires cargo available in path, install via rustup https://rustup.rs/")
+  endif()
+  execute_process(COMMAND ${CARGO} install cxxbridge-cmd --version 1.0.107)
+  find_program(CXXBRIDGE cxxbridge PATHS "$ENV{HOME}/.cargo/bin/")
+endif()
+
+emit_cxxrs_header()
+emit_cxxrs_for_module(everestrs)
+
+add_library(everestrs_sys STATIC
+  ${CMAKE_CURRENT_BINARY_DIR}/cxxbridge/everestrs/lib.rs.cc
+  everestrs_sys/everestrs_sys.cpp
+)
+
```

Comment:
Without this option the linker gives an error when building the module:
`set_property(TARGET everestrs_sys PROPERTY POSITION_INDEPENDENT_CODE ON)`